src/onInit/cleanData.js

```diff
@@ -15,6 +15,7 @@ export default function cleanData() {
                 nRemoved > 1 ? 's have' : ' has'
             } been removed.`
         );
+    this.initial_data = clean;
```

Comment:
How dare you use `chart.initial_data` instead of good ol' `chart.super_raw_data`? Kids these days.

Reply:
I'm not a super freak.